src/main/java/frc/robot/subsystems/intake/commands/BasicIntakeCommand.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems.intake.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.intake.Intake;
+
+/**
+ * Default intake command just sucks ball.
+ */
+public class BasicIntakeCommand extends CommandBase {

```

Comment:
1) There is no need to include "command" in a command name.
2) Is "basic" really helpful here?